docs/events/decentralized_event_handlers.md

```diff
@@ -0,0 +1,151 @@
+```python exec
+import reflex as rx
+```
+
+# Decentralized Event Handlers
+
+## Overview
+
+Decentralized event handlers allow you to define event handlers outside of state classes, providing more flexible code organization. This feature was introduced in Reflex v0.7.10 and enables a more modular approach to event handling.
+
+With decentralized event handlers, you can:
+- Organize event handlers by feature rather than by state class
+- Reuse event handlers across multiple state classes
```

Comment:
i dont think this applies... an event handler is still only bound to one state class, decentralized or no